Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt

```diff
@@ -68,7 +68,7 @@ data class VaccinatedPerson(
     }
 
     private fun getNewestFullDose(): VaccinationCertificate? = vaccinationCertificates
-        .filter { it.doseNumber == it.totalSeriesOfDoses }
+        .filter { it.doseNumber >= it.totalSeriesOfDoses }
```

Comment:
I just aligned with Martin Hermes ,whenever we have a booster vaccination , the vaccination should be considered `complete` and `immune`. in this it covering only `complete`

Reply:
I adjusted it, so a person is considered as immune when he/she gets a booster, so the info box should be gone now. Is this fine?
Please test it. I am currently unable to test myself, unfortunately.